x/ethbridge/client/rest/rest.go

```diff
@@ -18,23 +19,31 @@ import (
 )
 
 const (
-	restNonce          = "nonce"
-	restEthereumSender = "ethereumSender"
+	restEthereumChainID = "ethereumChainID"
+	restBridgeContract  = "bridgeContract"
+	restNonce           = "nonce"
+	restSymbol          = "symbol"
+	restTokenContract   = "tokenContract"
+	restEthereumSender  = "ethereumSender"
 )
 
 type createEthClaimReq struct {
-	BaseReq        rest.BaseReq `json:"base_req"`
-	Nonce          int          `json:"nonce"`
-	EthereumSender string       `json:"ethereum_sender"`
-	CosmosReceiver string       `json:"cosmos_receiver"`
-	Validator      string       `json:"validator"`
-	Amount         string       `json:"amount"`
+	BaseReq               rest.BaseReq `json:"base_req"`
+	EthereumChainID       int          `json:"ethereum_chain_id"`
+	BridgeContractAddress string       `json:"bridge_contract_address"`
+	Nonce                 int          `json:"nonce"`
+	Symbol                string       `json:"symbol"`
+	TokenContractAddress  string       `json:"token_contract_address"`
+	EthereumSender        string       `json:"ethereum_sender"`
+	CosmosReceiver        string       `json:"cosmos_receiver"`
+	Validator             string       `json:"validator"`
+	Amount                string       `json:"amount"`
 }
 
 // RegisterRoutes - Central function to define routes that get registered by the main application
 func RegisterRESTRoutes(cliCtx context.CLIContext, r *mux.Router, storeName string) {
 	r.HandleFunc(fmt.Sprintf("/%s/prophecies", storeName), createClaimHandler(cliCtx)).Methods("POST")
-	r.HandleFunc(fmt.Sprintf("/%s/prophecies/{%s}/{%s}", storeName, restNonce, restEthereumSender), getProphecyHandler(cliCtx, storeName)).Methods("GET")
+	r.HandleFunc(fmt.Sprintf("/%s/prophecies/{%s}/{%s}/{%s}/{%s}/{%s}/{%s}", storeName, restEthereumChainID, restBridgeContract, restNonce, restSymbol, restTokenContract, restEthereumSender), getProphecyHandler(cliCtx, storeName)).Methods("GET")
```

Comment:
mmmm I have my doubts with this endpoint. It doesn't look very RESTful. Any chance we could use query params 
or an alternative to avoid adding all the fields on the endpoint itself?

Reply:
https://github.com/cosmos/peggy/pull/68/commits/d191c68092829b52e59bfa2e00d70802873da1a9